### PR TITLE
Update feature-request issue template to move instructions into comments.

### PR DESCRIPTION
**Pull request by mklement0**
# PR Summary

Moved instructions into comments.

## PR Context

While users should _see_ instructions, they shouldn't be _included_ when the issue is created.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)

### CODE REVIEW

**Comment by vexx32** [comment]
@mklement for those users who don't know the markdown / html syntax all that well, might be worth mentioning that anything put inside the comment body won't be displayed and they should either remove them or put text outside the comments.

**Comment by mklement0** [comment]
That's a good idea, @vexx32, but I personally feel I've put in my time 😁

If someone wants to take that on, note that it should also be done for the other templates; it's missing from the bug template too.

It's also worth improving the templates that essentially say: "Do not create an issue here, just read this text and then take appropriate action elsewhere".

It sounds like preventing users from still creating an issue cannot be prevented for technical reasons, so the next best thing is to add a note at the top that shouts "DO NOT CREATE AN ISSUE - JUST READ".

File: .github/ISSUE_TEMPLATE/Feature_Request.md [hunk 1]
```diff
@@ -9,11 +9,15 @@ assignees: ''
```
Comment (RDIL) [inline .github/ISSUE_TEMPLATE/Feature_Request.md]: at the top
```suggestion
<!--anything that looks like this is a comment and cannot be seen after creating the issue.-->
```

**Review by iSazonov — APPROVED** [review]

**Comment by iSazonov** [comment]
@PoshChan rebuild macos

**Comment by PoshChan-Staging** [comment]
@iSazonov, I do not understand: rebuild macos

**Comment by PoshChan** [comment]
@iSazonov, I do not understand: rebuild macos

**Comment by iSazonov** [comment]
@PoshChan Please rebuild macos

**Comment by PoshChan** [comment]
@iSazonov, did not find matching build context: `PowerShell-CI-macOS`; allowed contexts: PowerShell-CI-windows, PowerShell-CI-linux, PowerShell-CI-static-analysis

**Comment by PoshChan-Staging** [comment]
@iSazonov, you are not authorized to request a rebuild

**Comment by iSazonov** [comment]
@SteveL-MSFT Could you please look the issue with PoshChan ?

**Comment by RDIL** [comment]
@iSazonov it’s a config issue

File: .github/ISSUE_TEMPLATE/Feature_Request.md (hunk 1 above)
Comment (TravisEz13) [reply .github/ISSUE_TEMPLATE/Feature_Request.md]: I find this actually quite annoying, but I know what they are....

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
@PoshChan 	Please remind me in 22 hours

**Comment by PoshChan** [comment]
@TravisEz13, will remind you in 22 hours

**Comment by PoshChan-Staging** [comment]
@TravisEz13, you are not authorized to request a reminder

**Comment by RDIL** [comment]
@iSazonov fixed in #9188

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 22 hours ago

**Comment by RDIL** [comment]
The reminder you were 'not authorized' to request 😆

**Comment by SteveL-MSFT** [comment]
@PoshChan Please remind me in 1 minute

**Comment by PoshChan-Staging** [comment]
@SteveL-MSFT, will remind you in 1 minute

**Comment by PoshChan** [comment]
@SteveL-MSFT, will remind you in 1 minute

**Comment by PoshChan** [comment]
@SteveL-MSFT, this is the reminder you requested 1 minute ago

**Comment by iSazonov** [comment]
@PoshChan Please rebuild macos

**Comment by PoshChan** [comment]
@iSazonov, did not find matching build context: `PowerShell-CI-macOS`; allowed contexts: PowerShell-CI-windows, PowerShell-CI-linux, PowerShell-CI-static-analysis

**Comment by iSazonov** [comment]
@PoshChan Please rebuild macOS

**Comment by PoshChan** [comment]
@iSazonov, could not find a matching build target

**Comment by iSazonov** [comment]
@PoshChan Please rebuild PowerShell-CI-macOS

**Comment by PoshChan** [comment]
@iSazonov, I do not understand the build target(s) `PowerShell-CI-macOS`; I only allow `all`, `windows`, `staging`, `linux`, `macos`

**Comment by iSazonov** [comment]
@PoshChan Please rebuild linux

**Comment by PoshChan** [comment]
@iSazonov, successfully started rebuild of `PowerShell-CI-Linux`

**Comment by iSazonov** [comment]
@SteveL-MSFT The bot can not rebuild macos if status is
> PowerShell-CI-macos Expected — Waiting for status to be reported 

Is it possible to fix this?

**Comment by PoshChan-Staging** [comment]
@SteveL-MSFT, this is the reminder you requested 1 minute ago

**Review by TravisEz13 — APPROVED** [review]

**Comment by SteveL-MSFT** [comment]
@iSazonov I'll have her respond with more tracing info to figure out why it failed.  Seems like it tried to query something from AzDevOps but in that state nothing got returned.

**Pull request merged**